### PR TITLE
Fix cli package/imports

### PR DESCRIPTION
**Pull request by jasonborg**
This changes fixes the cli package support and follows
https://google.github.io/styleguide/pyguide.html#23-packages

* Fix imports, each module is imported using the full pathname location
  of the module.
* Adds a __main__.py

The cli is now run from the root of the repo as:

```
python -m cli
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ Where PROJECT_ID is your project ID
 This will instruct the debugger CLI to create and use a database with the name
 `PROJECT_ID-cdbg`
 
-1. Run `python3 cli/cli.py init` in the cloned `snapshot-debugger`
+1. Run `python3 -m cli init` in the cloned `snapshot-debugger`
```
Comment (mctavish) [inline README.md:177]: Per offline discussion, cli is too much of a generic module name in this context.  It should either be namespaced or be given a more specific name for disambiguation.

File: cli/cli.py [hunk 2]
```diff
@@ -66,10 +66,3 @@ def main():
 
   # This will run the appropriate command.
   args.func(args=args, cli_services=cli_services)
-
-
-if __name__ == '__main__':
-  try:
-    main()
-  except SilentlyExitError:
-    sys.exit(1)
```
[the remark this answers is not in the document]
Comment (jasonborg) [reply cli/cli.py:75]: Good call thanks, I've added a message should a user attempt to run this file.

PTAL

File: README.md (hunk 1 above)
Comment (jasonborg) [reply README.md]: I've switched to using `snapshot_dbg_cli` as the package name.

PTAL

**Review by mctavish — CHANGES_REQUESTED** [review]
Make sure that you update the link on line 44 of the README to reflect the new path for the command reference.

I cannot figure out how to put in a comment on a line that you haven't changed.

File: README.md (hunk 1 above)
Comment (mctavish) [reply README.md]: LGTM

File: cli/cli.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mctavish) [reply cli/cli.py:75]: I like your approach.

**Review by mctavish — APPROVED** [review]

**Comment by jasonborg** [comment]
> Good catch about the Command reference, done.

**Pull request merged**